__tests__/articlesRoutes.tests.js

```diff
@@ -0,0 +1,42 @@
+const runSeed = require("../src/db/seeds/run-seed");
+const db = require("../src/db");
+const request = require("supertest");
+const app = require("../src/app");
+
+beforeEach(() => {
+  return runSeed();
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api/articles/:article_id", () => {
+  describe("Happy path", () => {
+    it("should return 200", async () => {
+      await request(app).get("/api/articles/1").expect(200);
+    });
+    it("shouold return a object matching the article schema", async () => {
```

Comment:
When requesting a specific article, we should be asserting exactly what the response object will look like based on the data rather than expecting any value based on type.

Reply:
Absolutely agree, was going a bit too quick. Will fix and merge this eve